CODE_OF_CONDUCT.md

```diff
@@ -13,44 +41,43 @@ We are committed to providing a friendly, safe and welcoming environment for all
 -   Gracefully accepting constructive criticism
 -   Focusing on what is best for the community
 -   Showing empathy towards other community members
+-   Helping others understand you
+-   Using spoiler warnings
+-   Using trigger warnings (for things like food, violence, queerphobia)
 
 ##### Examples of unacceptable behavior by participants include:
 
 -   The use of sexualized language or imagery and sexual attention or advances
 -   The use of inappropriate images, including in a community member's avatar
 -   The use of inappropriate language, including in a community member's nickname
 -   Any spamming, flaming, baiting or other attention-stealing behavior
--   Trolling, insulting/derogatory comments, and personal or political attacks
+-   Insulting/derogatory comments, and personal or political attacks
 -   Public or private harassment
 -   Publishing others' private information, such as a physical or electronic address, without explicit permission
--   Discussing topics that are overly polarizing, sensitive, or incite arguments. This includes the discussion of polarizing political views, violence, suicide, and rape.
--   Responding with “RTFM”, "just google it” or similar phrases in response to help requests
+-   Discussing topics that are overly polarizing, sensitive, or incite arguments outside of #the-forum threads.
+-   Responding with “RTFM”, "just google it” or similar phrases in response to help requests.
```

Comment:
I think in some very obvious cases this is fine imo, but i guess we can leave it here just in case someone does it to a toxic extent.

Reply:
and even then, only the "just google it" part. RTFM is not good.

Reply:
the general idea is that we'd rather have someone at least say "you could try googling these words, which mean this and that" to kinda teach how to google things? its also a skillset someone might not have - especially with google becoming worse and worse at actual results.

Reply:
:+1:

Reply:
yeah, ageed - "just google it" is not a generous thing to say

Reply:
Generally what I do is to say I googled it along with the answer I found and the link to it, which probably doesn't fall under "just google it” but still tells them that they could've done it that way